### PR TITLE
Add more documentation on customization

### PR DESCRIPTION
**Pull request by ehuelsmann**

### CODE REVIEW

File: part-customization.tex [hunk 1]
```diff
@@ -43,7 +43,156 @@ \section{Introduction}
...
+\section{Workflow configuration}
+
+LedgerSMB uses the concept of workflows\index{workflows} to manage the document life cycle.
+An invoice for example may be \texttt{saved}, \texttt{posted} or \texttt{reversed}.  These
+are states in the life cycle of the invoice.  Each state may have associated actions.  In
+case of the invoice an example may be to \texttt{Post} it when it's \texttt{saved} or to
+\texttt{E-mail} it when it's \texttt{posted}.
+
+Actions associated with a state may be available or not determined by one or more conditions.
+An examples are ``is the posting date of the transaction in a closed period''
```
Comment (neilt) [inline part-customization.tex:59]: Grammar, 'An examples' -> 'Examples'

File: part-customization.tex [hunk 2]
```diff
@@ -43,7 +43,156 @@ \section{Introduction}
...
+	and conditions) with the allowed transitions per state (actions).
+\end{enumerate}
+
+Customized versions of workflow definition files must be stored in
+ \texttt{./customized\_workflows/}\footnote{See
+ 	\secref{subsubsec-global-config-ledgersmb-yaml-workflows} on how to configure
+ these paths}.  Files stored in this location will be used to override the ones in
+the standard location \texttt{./workflows}.  No changes should be made directly to
+the standard files because of the risk that those files will be overwritten on upgrade
+-- loosing the changes.
```
Comment (neilt) [inline part-customization.tex:82]: How is are the standard workflow files disabled?  The use case is providing a custom workflow to replace a system provided workflow.

**Review by neilt — APPROVED** [review]
Excellent, I've attached some notes, but a big improvement.

File: part-customization.tex (hunk 1 above)
Comment (ehuelsmann) [reply part-customization.tex:59]: Done.

File: part-customization.tex (hunk 2 above)
Comment (ehuelsmann) [reply part-customization.tex]: Do you think the expanded explanation makes sense?

**Review by neilt — APPROVED** [review]
Looks good.

**Pull request merged**